spec/features/course/assessment/submission/guided_spec.rb

```diff
@@ -53,6 +48,15 @@
         expect(page).to have_selector('h2', text: mrq_questions.first.title)
 
         click_link '2'
+        # Students should not be able to go to Q2 before Q1 is done.
```

Comment:
Should this be a separate test? So there'll be a test for navigating between done questions, another to check that a student cannot jump ahead, then one to check that completing a question allows him to move on.